src/main/java/org/apache/accumulo/testing/continuous/BulkIngest.java

```diff
@@ -49,14 +49,15 @@
 
   @Override
   public int run(String[] args) throws Exception {
+    String ingestInstanceId = UUID.randomUUID().toString();
+
     Job job = Job.getInstance(getConf());
+    job.setJobName("BulkIngest_" + ingestInstanceId);
```

Comment:
I like having the uuid in the name because it could be useful for debugging if data loss were to happen.